programs/08-wasm-eth/src/contract.rs

```diff
@@ -0,0 +1,649 @@
+use std::convert::Into;
```

Comment:
Quite a lot of things to improve here, but I mostly added it to be able to test building and checking the wasm file to make sure I didn't have any bad dependencies and so on. It will be cleaned up and done proper in https://github.com/cosmos/solidity-ibc-eureka/issues/143

Reply:
That goes for all the files in programs/08-wasm-eth, really. It does seem to work pretty well already though :)